CONTRIBUTING.md

```diff
@@ -14,10 +14,16 @@ If you find a bug in this project, have trouble following the documentation, or
 - **Include system details** like what the browser, library or operating system you’re using and its version.
 - **Paste error output** or logs in your issue or in a [Gist](http://gist.github.com/). If pasting them in the issue, wrap it in three backticks: ` ``` ` so that it renders nicely.
 
+## Stack Overflow Questions and Documentation
+
+There are a number of situations where a particular approach to a ddev solution can be stated more easily in [Stack Overflow](https://stackoverflow.com/tags/ddev) (use the "ddev" tag). We respond there quick quickly, but if you know the answer already, create the question there and then click the checkbox at the bottom "Answer your own question". Stack Overflow is often the best place to incubate documentation that affects just a few people or that just needs time to get responses. And it's highly searchable on the web.
```

Comment:
Good additions, everything looks good to me except for one typo I spotted:

>We respond there **quick quickly**, but...

Other than that, 👍